### PR TITLE
Blog post celebrating 🎉 talks annoucement

### PR DESCRIPTION
**Pull request by miquelbeltran**
Merge to gh-pages once the schedule is ready.

Feel free to modify, don't ask: DO! 🌮

### CODE REVIEW

File: _posts/2016-10-10-devfest-talks-announcement.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Talks announcement!"
+date:   2016-11-1 23:59:59
+isStaticPost: false
+---
+
+Less than a two weeks left to our annual DevFest and work piles up on our side.
```
Comment (stefanhoth) [inline _posts/2016-10-10-devfest-talks-announcement.markdown]: > and work piles up on our side.

That sounds a bit too negative, even if true. 

I suggest "and we're on the last stretch to make it an awesome event yet again."

File: _posts/2016-10-10-devfest-talks-announcement.markdown [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Thankfully, we have an awesome community that submitted great talks for Android, Web, Golang and Cloud.
+It has been an intense task to do, but finally we came with our selection of talks and workshops.
+
+We are happy to announce that we will have FIVE (you read it right) workshops on Friday.
+And on Saturday, more than twenty full length talks for all kinds of Google technologies.
+
+Head over here for the full [schedule](http://2016.devfest-berlin.de/schedule/).
```
Comment (stefanhoth) [inline _posts/2016-10-10-devfest-talks-announcement.markdown]: [the full schedule] 

don't make the link target unnecessarily small.

File: _posts/2016-10-10-devfest-talks-announcement.markdown [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Can
```
Comment (stefanhoth) [inline _posts/2016-10-10-devfest-talks-announcement.markdown]: ??? :-)

File: _posts/2016-10-10-devfest-talks-announcement.markdown [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+We hope you are excited just as we are, and to see you Friday 11th at ImmobilienScout24!
+
+Also: we still have tickets left, grab them while you can [here](http://2016.devfest-berlin.de/tickets/)
```
Comment (stefanhoth) [inline _posts/2016-10-10-devfest-talks-announcement.markdown]: [grab them while you can here]

**Review by stefanhoth — COMMENTED** [review]
I won't block it but I think at least the stray word need to be fixed.

File: _posts/2016-10-10-devfest-talks-announcement.markdown (hunk 3 above)
Comment (mariobodemann) [reply _posts/2016-10-10-devfest-talks-announcement.markdown]: Damn Android keyboard  ... ;)

**Pull request merged**